Automattic/jetpack
# Donations Block: do not use admin color on the frontend

Fixes #30811

## Proposed changes:

Follow-up from #16687

The Donations block uses a common stylesheet that is used in the block editor and on the frontend, where css vars like `--wp-admin-theme-color` are not available. Let's switch to using a color that will be available on the frontend.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site that's connected to WordPress.com.
* Install the Kadence theme
* Go to Posts > Add New and add a Donations block.
* Make sure you can switch between tabs for donation frequency, and that the text is readable.
* **Make sure this works on the frontend as well.**
* Try with a different theme like Twenty Twenty Three.
